src/main/java/core/basesyntax/figures/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figures;
+
+public interface Drawable {
+    String draw();

```

Comment:
seems you've missed the task requirement a bit:
`To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this)`

`draw()` method should not return anything - it should simply print the information about a figure
```suggestion
    void draw();

```